modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -175,22 +175,29 @@ Avro::
 | string | string
 | record | struct
 | array | list
-| maps | list
-| fixed | fixed
+| maps | map
+| fixed | fixed*
 | decimal | decimal
-| uuid | uuid
+| uuid | uuid*
 | date | date
-| time | time
+| time | time*
 | timestamp | timestamp
 |===
 
+*These types are not currently supported in Unity Catalog managed Iceberg tables.
+
+There are some cases where the Avro type does not map directly to an Iceberg type and Redpanda applies the following transformations:
```

Comment:
@rockwotj @mattschumpert We received a [docs issue](https://redpandadata.atlassian.net/browse/DOC-1178) that points out some confusion with this section of the page. Are these changes on the right track to resolve the issue or do I need to make other changes?